modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/GridCacheQueryRequest.java

```diff
@@ -698,6 +711,11 @@ public Boolean isDataPageScanEnabled() {
 
                 writer.incrementState();
 
+            case 26:
```

Comment:
It looks like, you added this code manually. Unfortunately, it does not work and might lead to compatibility issues. The `MessageCodeGenerator` should be used instead to generate such code. [GG8 secret knowledge :smiling_imp: ]

Reply:
Wow, thanks!